public/instructors_list.php

```diff
@@ -153,9 +163,9 @@
         Course Lecture: 
         <select Name="lecture_selection" id="lecture_selection">
             <option value="">----Select----</option>
-        <?php foreach($lecture as $row) { ?>
-            <option value="<?php echo $row->id; ?>">
-            <?php echo $row->id; ?>
+        <?php foreach($unassigned_lectures as $row) { ?>
+            <option value="<?php echo $row->lecture_code; ?>">
```

Comment:
I think this should be `<option value="<?php echo $row->id; ?>"`? The lecture table doesn't have a lecture_id field.

Reply:
done